### PR TITLE
Improve `_ArgInfo` init

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

File: cupy/core/_kernel.pxd [hunk 1]
```diff
@@ -35,6 +35,15 @@ cdef class _ArgInfo:
...
         readonly bint index_32_bits
 
+    cdef _ArgInfo _init(
+            self,
+            _ArgKind arg_kind,
+            type typ,
+            object dtype,
+            int ndim,
+            bint c_contiguous,
+            bint index_32_bits)
```
Comment (asi1024) [inline cupy/core/_kernel.pxd:45]: Could you fix indentation for Travis failure?

Comment (okuta) [reply cupy/core/_kernel.pxd]: I fixed.

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**